analysis/topEFT/datacard_maker.py

```diff
@@ -334,7 +337,13 @@ def getHist(d_hists,name):
             xmin = h.GetXaxis().GetXmin()
             xmax = h.GetXaxis().GetXmax()
             xwidth = h.GetXaxis().GetBinWidth(1)
-            h.GetXaxis().SetRangeUser(xmin, xmax + 1.5*xwidth) #Include overflow bin in ROOT
+            # Fold overflow bin in ROOT into last bin (combine does NOT use the overflow bin)
+            last = h.GetBinContent(h.GetNbinsX())   # last bin
+            over = h.GetBinError(h.GetNbinsX())     # overflow
+            e_last = h.GetBinError(h.GetNbinsX())   # last bin err
+            e_over = h.GetBinError(h.GetNbinsX()+1) # overflow error
+            h.SetBinContent(h.GetNbinsX(), last+over) # Add overflow to last bin
+            h.SetBinError(h.GetNbinsX()+1, (last**2+over**2)**.5) # Add overflow error in quadrature to last bin
```

Comment:
I ended up running into a few potential issues while testing this. There were a few minor ones, e.g. `h.GetNbinsX()` -> `h.GetNbinsX()+1` and `GetBinError`->`GetBinContent`, but also a potentially more complicated one, where the fact that `getHist()` is called multiple times on the same hist meant that the overflow was getting added to the last bin more than once. Since these updates are not completely related to this PR to begin with, I thought it might be cleaner to address the issues in a different branch, so I've created the branch and made a PR [here](https://github.com/TopEFT/topcoffea/pull/170). I think that PR is independent of this one, so once it has been reviewed, I think it'd make sense to merge it into the master (as opposed to merging it into this branch).